config/sync/core.entity_view_display.node.frontpage.default.yml

```diff
@@ -0,0 +1,108 @@
+uuid: 701724ea-2fb1-4797-bc3c-4d5511f2cd0e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.frontpage.body
+    - field.field.node.frontpage.field_image
+    - node.type.frontpage
+  module:
+    - image
+    - layout_builder
+    - layout_discovery
+    - text
+    - user
+third_party_settings:
+  layout_builder:
+    allow_custom: false
+    enabled: true
+    sections:
+      -
+        layout_id: layout_onecol
+        layout_settings:
+          label: Hero
+        components:
+          7f3a6318-09e5-44ca-b664-857f40f01db4:
+            uuid: 7f3a6318-09e5-44ca-b664-857f40f01db4
+            region: content
+            configuration:
+              label_display: '0'
+              context_mapping:
+                entity: layout_builder.entity
+              id: 'field_block:node:frontpage:field_image'
+              formatter:
+                label: above
+                settings:
+                  image_style: ''
```

Comment:
I see you haven't set an image_style for the Front/Landing page hero image. Imagine two scenarios:

- the user uploads a picture that is too small (e.g. 500x500)
- the user uploads a picture that is too large (e.g. 5000x5000)

As this image is going to be blurred and probably appear as a 'background' it might not cause issues visually, but setting up an image style that scales and crops the image certainly helps large images to be smaller in bytes and improve performance.